.github/workflows/test.yml

```diff
@@ -5,15 +5,15 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.20.x,1.21.x]
+        go-version: [1.20.x,1.21.x,1.22.x]
```

Comment:
Perhaps `^1` to use the latest cached version?

Reply:
What do you mean, have a single `^1.20.x` for the latest cached version or to do that across the matrix range?

In the past, `go-billy` broke on Windows for specific Go minors, so it would be good to ensure we keep the last 3 minors.

Reply:
But then we'd need to constantly update the workflows. Instead, we can use `[^1, ...]` and whatever specific Go versions we need to test against (i.e. the ones that broke in the past)

Reply:
Ok, let's give it a go on a follow-up PR.